vagrant/ansible/vagrant-provision.yaml

```diff
@@ -86,20 +73,27 @@
         - net.bridge.bridge-nf-call-ip6tables
         - net.bridge.bridge-nf-call-iptables
 
-    - name: Partition and format gravity disk
-      shell: |-
-        # KVM disk
-        if [ -b /dev/vdb ]; then
-          drive=/dev/vdb
-        # VirtualBox disk
-        elif [ -b /dev/sdb ]; then
-          drive=/dev/sdb
-        else
-          echo "No /dev/vdb nor /dev/sda found" 2>&1
-          exit 1
-        fi
-        parted -a opt "$drive" mktable msdos
-        parted -a opt "$drive" mkpart primary ext4 0% 100%
-        mkfs.ext4 -L GRAVITY "${drive}1"
-        mkdir /var/lib/gravity
-        mount LABEL=GRAVITY
+    - name: Find root disk
+      shell: mount -l | grep ' on / type ' | head -n1
+      register: root_mount
+
+    - set_fact:
+        partition_path: "{{ root_mount.stdout.split(' ')[0] }}"
+        disk_fstype: "{{ root_mount.stdout.split(' ')[4] }}"
+    - set_fact:
+        disk_path: "{{ partition_path | regex_search('(/dev/\\w+)\\d+', '\\1') | first }}"
+        partition_number: "{{ partition_path | regex_search('/dev/\\w+(\\d+)', '\\1') | first }}"
+    - debug:
+        msg: "partition_path: {{ partition_path }} disk_path: {{ disk_path }} fs_type: {{ disk_fstype }} partition: {{ partition_number }}"
+
+    - name: Grow partition until maximum
+      shell: "growpart /dev/sda {{ partition_number }}"
```

Comment:
What's the `disk_path` for then if this defaults to `/dev/sda` - it could default to `/dev/sda1` as the first partition as well.